easy_phi/app.py

```diff
@@ -471,65 +493,70 @@ def get(self):
 
     post = put = delete = get
 
-if __name__ == '__main__':
-    # parse command line twice to allow pass configuration file path
-    parse_command_line(final=False)
-    parse_config_file(options.conf_path, final=False)
-    # command-line args override conf file options
-    parse_command_line()
-else:
-    try:
+
```

Comment:
This change involves moving a lot of code to fix very minor issue. Before, configuration import was a part of script that was executed on import. Depending on how module was imported, we could get configuration from command line only or combined from cmdline and configuration file. Application settings include part of configuration, so if application was imported from other module instead of launching as a console script, we might loose some settings.

Yet we only have only one use case to face this issue (unit test), in future we might expect it to happen again in other scenarios. For long term maintainability, it is better to make this change now